src/components/Wrapper.js

```diff
@@ -38,8 +44,13 @@ export const WrapperRow = styled.View`
 
 export const WrapperWrap = styled(WrapperRow)`
   flex-wrap: wrap;
-  justify-content: space-between;
   width: 100%;
+
+  ${(props) =>
+    props.spaceBetween &&
+    css`
+      justify-content: space-between;
+    `};
```

Comment:
as `WrapperRow` already has a `spaceBetween` prop and `WrapperWrap` is based on it, the prop does not need to be declared here again.

Reply:
done in e7c8e34880bbaf13da77dedcb934806cd58899c5